.mailmap

```diff
@@ -116,6 +116,7 @@ Mathis Michel <mathis.michel@outlook.de>
 Michael Crosby <crosbymichael@gmail.com> <michael@thepasture.io>
 Michael Katsoulis <michaelkatsoulis88@gmail.com>
 Michael Zappa <michael.zappa@gmail.com> <michaelzappa@microsoft.com>
+Michael Zappa <michael.zappa@gmail.com> <michaelzappa@Michaels-MacBook-Pro-8.local>
```

Comment:
good laptop

Reply:
We can ditch these later after a release, many entries here were not intentional